### ISSUE TITLE
Data endpoint requires autometa=true

### ISSUE THREAD

**Issue by nguyer** [hyperledger/firefly#205]
Right now, if someone attempts to upload a file to the `/namespaces/{namespaceId}/data` endpoint, and only sets the `file` field, the request will fail with the following response:

```
HTTP 500 Internal Server Error
```
```json
{
    "error": "FF10116: Database insert failed: NOT NULL constraint failed: data.value"
}
```

As a workaround it is possible to get this to work by adding another field to the form called `autometa` and setting its value to be `true`. This upload should work without needing to set this field though.

**Comment by peterbroadhurst** [comment]
You can also set `metadata` to be any valid JSON

**Comment by peterbroadhurst** [comment]
Fixed in #207 to allow a `null` value
